Source/cryptography/implementation/SecApi/VaultNetflix.cpp

```diff
@@ -19,7 +19,7 @@
 
 #include "../../Module.h"
 #include "Vault.h"
-
+#define MAX_FILE_SIZE 300
```

Comment:
Th pre-processor is not our best friend :-) It does not adhere to namespaces for example. Do like the fact that you extracted the magicals from the code but could you make this magical a:
_static constexpr uint32_t MaxFileSize = 300;_
This is more inline with the coding guidlines

Reply:
Change Done.